### PR TITLE
Mention contract demos in hack/chains readme

### PR DESCRIPTION
**Pull request by simonbaird**

### CODE REVIEW

**Review by lcarva — APPROVED** [review]
Nice!

File: hack/chains/README.md [hunk 1]
```diff
@@ -157,3 +157,31 @@ your Argo CD is in sync.
...
+See the comments in that script for more details on how to use it.
+
+### End to end demo
+
+This will run a build pipeline, then run a QE pipeline, and then use the above
+release pipeline demo to verify that build against the Enterprise Contract.
+
+    ./end-to-end-demo.sh
+
+There is a [video showing this in action](https://drive.google.com/file/d/1DEqAVhqNhu2L1t8w3_fkxAY860YOnx6M/view?usp=sharing)
```
[the remark this answers is not in the document]
Comment (simonbaird) [reply hack/chains/README.md:186]: Should consider if there is anything sensitive in there. Don't want infosec to send a swat team to Zoran's house..

[the remark this answers is not in the document]
Comment (simonbaird) [reply hack/chains/README.md:186]: Url could be changed in a future PR of course..

[the remark this answers is not in the document]
Comment (lcarva) [reply hack/chains/README.md:186]: +1, let's not block on this.

**Comment by simonbaird** [comment]
I wonder if the test is being skipped since it's a markdown only patch.

**Pull request merged**